WikiDrips/WikiTableViewController.swift

```diff
@@ -76,26 +116,36 @@ class WikiTableViewController: UITableViewController, UISearchResultsUpdating {
         let wikiDoc = wikiDocs[indexPath.section][indexPath.row]
         wikiCell.wikiTitleLabel?.text = wikiDoc.title
         wikiCell.wikiDateLabel?.text = dateFormatter.string(from: wikiDoc.date)
-        
-        guard let initials = wikiDoc.imageInitials, let rectangle = wikiCell.wikiTitleImageView?.bounds else {
-            return wikiCell
-        }
-        
-        let cacheKey:NSString = initials as NSString //NSCache won't take String, so casting to NSString
-        
-        if let cachedImage = imageCache.object(forKey: cacheKey) {
-            wikiCell.wikiTitleImageView?.image = cachedImage
+        if let image = wikiDoc.image {
+            print("From WikiDoc")
+            wikiCell.wikiTitleImageView?.image = image
         } else {
-            DispatchQueue.global().async {
-                let generatedImage = UIImage.image(withInitials: initials, in: rectangle)
-                DispatchQueue.main.async() { [weak self] in
-                    wikiCell.wikiTitleImageView?.image = generatedImage
-                    self?.imageCache.setObject(generatedImage, forKey: cacheKey)
-                }
-            }
+            wikiCell.wikiTitleImageView?.image = nil
+            generateImage(forItemAtIndex: indexPath)
```

Comment:
I would try to only generate images from the `UITableViewDataSourcePrefetching` methods and instead just show a default image here.